cmd/api/src/api/middleware/middleware.go

```diff
@@ -147,6 +148,19 @@ func ContextMiddleware(next http.Handler) http.Handler {
 	})
 }
 
+func parseUserIP(r *http.Request) string {
+	res := ""
+	if ipAddress := r.Header.Get("X-Forwarded-For"); ipAddress != "" {
+		res += "X-Forwarded-For: " + ipAddress + "; "
+	} else {
+		log.Warnf("No data found in X-Forwarded-For, possible upstream misconfig?")
```

Comment:
should this log include the remote addr and /or request endpoint, etc so there's additional info that can discern one of these from each other?